packages/synthetix-main/contracts/modules/core/CollateralModule.sol

```diff
@@ -39,19 +39,21 @@ contract CollateralModule is
         uint liquidationReward,
         bool enabled
     ) external override onlyOwner {
-        if (!_collateralStore().collaterals.contains(collateralType)) {
-            // Add a collateral entry
-            _collateralStore().collaterals.add(collateralType);
+        CollateralStore storage store = _collateralStore();
+        SetUtil.AddressSet storage collaterals = store.collaterals;
+
+        if (!collaterals.contains(collateralType)) {
+            collaterals.add(collateralType);
         }
 
-        CollateralData storage collateralData = _collateralStore().collateralsData[collateralType];
+        CollateralData storage collateral = store.collateralsData[collateralType];
```

Comment:
This is the storage read improvement.

In general, when something is read from storage multiple times, (e.g. `_collateralStore()`), it's better both for readability and gas, to store the pointer as a variable.